tests/e2e/tools/README.md

```diff
@@ -1 +1,5 @@
 The `remove-containers` script will remove all containers (and images) which name start with node* or control from the output of podman ps. Useful to cleanup environment before executing tests and measures.
+
+### Freedom From Interference
+
+Under FFI c binaries build in different [automotive-repo](https://gitlab.com/CentOS/automotive/container-images/-/tree/main/images/ffi-tools)
```

Comment:
not sure if I follow the 'c' here in this sentence.

Reply:
Ack, typo